### PR TITLE
Pass global options to standard

### LINKED ISSUE #1: Add way to pass globals to standard
Currently there are no ways to pass general options to the standard linter.  All options are specifically picked from the options object.  One option that would be useful to send to standard is `globals`.  Right now, you can "pass" it to standard by having it in your package.json, but this prevents addons from adding globals.

### PR DESCRIPTION
**Pull request by jhitchins88**
Fixes #1

Adds the `global` or `globals` option to the `internalOptions` object, allowing someone to pass a globals array to the standard linter.  This will not prevent the `package.json` of the main file from adding its own `globals` options on top of this option.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -51,7 +51,8 @@ var StandardValidationFilter = function (inputNode, options = {}) {
     format: options.format ? options.format : undefined,
     throwOnError: options.throwOnError ? options.throwOnError : false,
     logOutput: options.logOutput ? options.logOutput : false,
-    fix: options.fix ? options.fix : false
+    fix: options.fix ? options.fix : false,
+    globals: options.globals || options.global
```
Comment (arschmitz) [inline lib/index.js:55]: why accept both the singular and plural?

**Review by arschmitz — CHANGES_REQUESTED** [review]

File: lib/index.js (hunk 1 above)
Comment (jhitchins88) [reply lib/index.js:55]: To mimic standard accepting of both: https://github.com/standard/standard#i-use-a-library-that-pollutes-the-global-namespace-how-do-i-prevent-variable-is-not-defined-errors

Comment (arschmitz) [reply lib/index.js:55]: If you have both in your package.json do they use both to standard how do they handle that? are they merged? does one take precedence?

Comment (jhitchins88) [reply lib/index.js:55]: It looks like `globals` takes precedence over `global`

https://github.com/standard/standard-engine/blob/master/index.js#L157

**Pull request closed without merging**